### PR TITLE
feat: add financial aid page

### PR DESCRIPTION
**Pull request by j8xixo12**

### CODE REVIEW

**Comment by j8xixo12** [comment]
Does someone can help to add shushi as a reviewer?

**Review by rockleona — APPROVED** [review]
LGTM!

File: contents/financial-aid.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Reimbursement
+
+sciwork 2024 Conference will send you a KKTIX discount code, which you can use to purchase tickets on KKTIX.
+
+---
+
+## Deadline
+
+From now until **December 13, 2024, at 23:59 (Taipei time)** is the application deadline.
```
Comment (YenPeiChen07) [inline contents/financial-aid.mdx]: The application deadline is not same as application form, which is on `December 13, 2024`.

image

**Review by YenPeiChen07 — COMMENTED** [review]
@j8xixo12, everything looks good. But I found that the deadline is not matched to the application form.

File: contents/financial-aid.mdx (hunk 1 above)
Comment (j8xixo12) [reply contents/financial-aid.mdx]: Thanks @YenPeiChen07 I have corrected the deadline in [db95704](https://github.com/sciwork/swconf2024/pull/22/commits/db95704019a03450029bad1ad350eddb544ccb13) !

**Review by YenPeiChen07 — APPROVED** [review]
LGTM.

**Pull request merged**